app/src/main/java/com/weatherxm/ui/photoverification/gallery/PhotoGalleryActivity.kt

```diff
@@ -125,7 +149,7 @@ class PhotoGalleryActivity : BaseActivity() {
                     analytics.trackEventUserAction(
                         AnalyticsService.ParamValue.START_UPLOADING_PHOTOS.paramValue
                     )
-                    navigator.showPhotoUpload(this, model.device, model.getPhotosLocalPaths())
```

Comment:
This function `getPhotosLocalPaths` isn't used anymore, so please delete it from the View Model and the respective unit tests.